Rover/Parameters.cpp

```diff
@@ -652,7 +652,7 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
 #if HAL_TORQEEDO_ENABLED
     // @Group: TRQD_
     // @Path: ../libraries/AP_Torqeedo/AP_Torqeedo.cpp
-    AP_SUBGROUPINFO(torqeedo, "TRQD_", 49, ParametersG2, AP_Torqeedo),
```

Comment:
Hi, this commit should also be prefixed with "Rover:".  I think we should shorten the parameter prefix to just "TRQ" because then the instance number will be added making it TRQ1 and TRQ2 which seems fine to me.   I think we try to keep our parameter prefixes to less than 4 these days.